starter-code/index.js

```diff
@@ -1,10 +1,46 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.items = [],
```

Comment:
Mistake was made here... Object LITERALS end with commas on each line BUT Object Constructors end with Semicolons. You added commas 🧐